app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -286,7 +287,7 @@ private fun MessageComposer(
                 // we get the effect of overlapping it
                 if (attachmentOptionsVisible) {
                     AttachmentOptions(
-                        onAttachmentPicked = onAttachmentPicked,
+                        onAttachmentPicked = { onAttachmentPicked(it, messageComposerState.getSelfDeletionTime().value) },
```

Comment:
By having `onAttachmentPicked(it, messageComposerState.getSelfDeletionTime().value)` inside `{..}` wouldn't it be prone to recomposition? not really sure if it has any harm buut, asking just in case 🤷🏻‍♂️

Reply:
I tested it and it seems to be working fine, thanks for the suggetions <3